### DISCUSSION TITLE
最新版本阿里云盘OPEN报错

### DISCUSSION THREAD

**Discussion by AspirantH** [AlistGo/alist#4845] [General]
failed get objs: failed to list objs: Post "https://openapi.aliyundrive.com/adrive/v1.0/openFile/list": dial tcp: lookup [openapi.aliyundrive.com](http://openapi.aliyundrive.com/) on [**1]:53: read udp [**1]:49224->[**1]:53: read: connection refused
![20230725001805](https://github.com/alist-org/alist/assets/93640658/31009d09-f56e-4674-a841-b020931e1b21)

**Comment by AspirantH** [comment]
奇怪，又自己自动恢复了。

**Comment by SuperKenVery** [comment]
我也遇到了……
